src/bp/ui-shared/src/Textarea/index.tsx

```diff
@@ -0,0 +1,48 @@
+import cx from 'classnames'
+import React, { FC, useEffect, useRef, useState } from 'react'
+
+import style from './style.scss'
+import { TextareaProps } from './typings'
+
+const Textarea: FC<TextareaProps> = props => {
+  const [forceUpdate, setForceUpdate] = useState(false)
+  const inputRef = useRef<HTMLTextAreaElement>(null)
+
+  useEffect(() => {
+    if (inputRef.current) {
+      inputRef.current.style.height = '0'
```

Comment:
can we play with classnames instead of injecting style ? I'm not a UI master bit this feels a bit odd ... There's probably a reason behind you know better than me can you explain

Reply:
We need to set the height after with ```inputRef.current.style.height = inputRef.current.scrollHeight + 'px'``` anyway so there's no real gain to have a class that we would set and then remove. This is how all the libraries to have an automatically growing textarea do it